File: Demo_template/lib/impl/native/win/com/codename1/ext/filechooser/FileChooserNativeImpl.cs
namespace com.codename1.ext.filechooser{

using System;
using System.Windows;

public class FileChooserNativeImpl {
    public bool isSupported() {
        return false;
    }

    public bool showNativeChooser(string accept, bool multi) {
        return false;
    }

}
}

File: codename-project-full-code-source-main/lib/impl/native/javase/com/codename1/ext/filechooser/FileChooserNativeImpl.java
package com.codename1.ext.filechooser;

public class FileChooserNativeImpl implements com.codename1.ext.filechooser.FileChooserNative{
    public boolean isSupported() {
        return true;
    }
    
    public boolean showNativeChooser(String accept, boolean multi) {
        return false;
    }

}
